src/components/mistakes/mistakes.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Mistakes = (props) => {
+  const {count} = props;
+
+  const mistakes = new Array(count).fill(``);
+
+  return (
+    <div className="game__mistakes">
+      {mistakes.map((it, i) => <div key={`mistake-${i}`} className="wrong" />)}
+    </div>
+  );
+};
+Mistakes.propTypes = {
+  count: PropTypes.number.isRequired,
+};
+export default Mistakes;
```

Comment:
так читается лучше 😊
```suggestion
const Mistakes = ({ count }) => {
  const mistakes = new Array(count).fill(``);

  return (
    <div className="game__mistakes">
      {mistakes.map((it, i) => <div key={`mistake-${i}`} className="wrong" />)}
    </div>
  );
};

Mistakes.propTypes = {
  count: PropTypes.number.isRequired,
};

export default Mistakes;
```